src/ripple/nodestore/impl/DatabaseShardImp.h

```diff
@@ -297,6 +326,38 @@ class DatabaseShardImp : public DatabaseShard
 
     void
     removeFailedShard(std::shared_ptr<Shard> shard);
+
+    // Returns the index that represents the logical
+    // partition between historical and recent shards
+    std::uint32_t
+    getShardBoundaryIndex(std::lock_guard<std::mutex> const&);
+
+    std::uint32_t
+    countHistoricalShards(std::lock_guard<std::mutex> const&);
```

Comment:
Base on this and another renaming suggestion you made, it seems that the convention is to came the method after the variable returned, rather than the action its performing - is that the case?

Reply:
Done